Element84/filmdrop-ui
# Ba/feature/add config setting for e84 logo

**Related Issue(s):**

- NA

**Proposed Changes:**

1. Add config var for `SHOW_BRAND_LOGO`
2. Update example config to include `SHOW_BRAND_LOGO`
3. Add new var info to readme
4. Update mock for tests
5. Fix prettier issue with new version (and add back in check for html)

### To test

- edit local `config.json` file to include `"SHOW_BRAND_LOGO": true`
- run app
- confirm logo renders in top right of UI
- stop app
- change local `config.json` file to be `"SHOW_BRAND_LOGO": false`
- run app
- confirm logo does not render in top right of UI
- stop app
- change local `config.json` file to include a URL for `DASHBOARD_BTN_URL` like `"DASHBOARD_BTN_URL": "https://www.google.com"`
- run app
- confirm dashboard button renders correctly with and without brand logo showing

**unit tests pass** 🍏 

**PR Checklist:**
